### ISSUE TITLE
Companion (positional and keyword) arguments missing

### ISSUE THREAD

**Issue by lfaucheux** [mechmotum/cyipopt#156]
Thx for this time-saving [`~.scipy_interface.IpoptProblemWrapper`](https://github.com/mechmotum/cyipopt/blob/master/cyipopt/scipy_interface.py). I guess that instead of 

```
# [...]
    def hessian(self, x, lagrange, obj_factor):
        H = obj_factor * self.obj_hess(x)  # type: ignore
        [...]
```
at line [160](https://bit.ly/3Kql8eC), you meant

```
# [...]
    def hessian(self, x, lagrange, obj_factor):
        H = obj_factor * self.obj_hess(x, *self.args, **self.kwargs)  # type: ignore
        [...]
```

?

**Comment by moorepants** [comment]
Likely, would you like to submit a pull request with the change?

**Comment by moorepants** [comment]
This has been fixed in #197
